src/Authenticator.ts

```diff
@@ -131,10 +131,57 @@ export class Authenticator{
             throw error
         }
     }
+
+    async changeActiveLicense(license: string){
+
+        const { clientId: ClientId, userId: UserId } = parseLicense(license)
+
+        try {
+            await this._changePassportMap({ ClientId, UserId })
+            await this._removeIdentity()
+            return license
+        } catch (err) {
+            const error: any = new Error(err)
+            throw error
+        }
+    }
+
+    async _changePassportMap(data: { ClientId: number; UserId: number }){
```

Comment:
Hmm, PascalCase? For å løse body til ChangePassportMap? Kanskje gjort dette opplegget i `mapToWWWEncoded`istedenfor?

```suggestion
    async _changePassportMap(data: { clientId: number; userId: number }){
```

Reply:
mhm.. testet i websiten nå, og den sender undefined på begge to her.. så her er det noe galt

Reply:
Ja, har fyrt opp alle dev pakkene i website accounting liksom.. men jeg er litt usikker på om det er denne linjen som ikke funker 
`const { clientId: ClientId, userId: UserId } = parseLicense(license)` eller om det er noe med license som kommer inn her.. Må se på det etter refinement

Reply:
åja, det kan nok stemme. For når du har det slik; 

```javascript
Object.fromEntries(
  licenseString
    .split(';')
    .map(section => section.split(':'))
)
```
Så sier du egentlig at den forventer en licenseString av typen `identityId:9d02dae3-1e06-48bc-9a5b-47c77e8888a2;clientId:824205686917063;userId:101010`. I dette tilfelle så er den heller `9d02dae3-1e06-48bc-9a5b-47c77e8888a2;824205686917063;101010`

Reply:
Kan jo kanskje bare gjøre sånn her da
`[_, ClientId, UserId] = "9d02dae3-1e06-48bc-9a5b-47c77e8888a2;824205686917063;101010".split(';')`

Reply:
For meg er det greit at det er PascalCase nå, kan endres senere da det ikke har noe betydning for logikk.